### PR TITLE
Add event app

### PR DESCRIPTION
**Pull request by redmyt**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [x] @makarman
- [x] @noobhunters
- [x] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue
    Added event model
    Added migration file
    Changed settings file
    Added docstrings

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/event/models.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+        :param budget: The money amount required for the event.
+        :type budget: integer
+
+        :param status: The number that points to the certain.
+                       stage of the event processing.
+        :type status: integer
+    """
+
+    STATUS_CHOISES = (
```
Comment (skramar) [inline eventually/event/models.py]: Choices

File: eventually/event/views.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"""
+Views module
+===========
+"""
+
+# from django.shortcuts import render
```
Comment (skramar) [inline eventually/event/views.py:6]: Do you need commented lines in git repository?

Comment (redmyt) [reply eventually/event/views.py:6]: These comments occur only for successful travis-pylint verification.

File: eventually/event/models.py [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+        :return: event name, event description, event status
+        """
+
+        return "{} {} {}".format(self.name, self.description, self.status)
+
+    def to_dict(self):
+        """
+        Method that converts event object to dictionary.
+
+        :return: dictionary with event's information
```
Comment (lhalam) [inline eventually/event/models.py:89]: add example

File: eventually/event/models.py [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+        :type event_id: integer
+
+        :return: event object or None if event does not exist
+        """
+
+        try:
+            event = Event.objects.get(id=event_id)
+            return event
+        except Event.DoesNotExist:
+            return None
```
Comment (lhalam) [inline eventually/event/models.py]: maybe `pass`

File: eventually/event/models.py [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+        event = Event()
+        event.name = name
+        event.description = description
+        event.start_at = start_at
+        event.duration = duration
+        event.longitude = longitude
+        event.latitude = latitude
+        event.budget = budget
+        if status:
```
Comment (lhalam) [inline eventually/event/models.py]: ???

File: eventually/eventually/settings.py [hunk 6]
```diff
@@ -31,6 +31,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'event',
```
Comment (lhalam) [inline eventually/eventually/settings.py]: your app after djandoApp

**Review by lhalam — CHANGES_REQUESTED** [review]

File: eventually/event/models.py (hunk 1 above)
Comment (redmyt) [reply eventually/event/models.py]: fixed

File: eventually/event/models.py (hunk 3 above)
Comment (redmyt) [reply eventually/event/models.py:89]: added

File: eventually/event/models.py (hunk 4 above)
Comment (redmyt) [reply eventually/event/models.py]: used pass

File: eventually/event/models.py (hunk 5 above)
Comment (redmyt) [reply eventually/event/models.py]: removed that `if` statement

File: eventually/eventually/settings.py (hunk 6 above)
Comment (redmyt) [reply eventually/eventually/settings.py]: fixed

**Review by Silent777 — DISMISSED** [review]

File: eventually/event/models.py [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+        |    'budget': 100,
+        |    'status': 0
+        | }
+        """
+
+        return {
+            'id': self.id,
+            'name': self.name,
+            'description': self.description,
+            'start_at': self.start_at,
```
Comment (lhalam) [inline eventually/event/models.py]: convert to timestamp

**Review by Silent777 — DISMISSED** [review]

File: eventually/event/models.py (hunk 7 above)
Comment (redmyt) [reply eventually/event/models.py]: added for all date fields

**Review by lhalam — DISMISSED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**